templates/nro/.circleci/config.yml.tmpl

```diff
@@ -23,6 +23,8 @@ job_environments:
     WP_DB_NAME: {{ .Env.MYSQL_USERNAME }}_{{ .Env.MYSQL_DATABASE }}_develop
     WP_STATELESS_BUCKET: {{ .Env.CONTAINER_PREFIX }}-stateless-develop
 {{- if isTrue .Env.MAKE_RELEASE }}
+  release_build_env: &release_build_env
+    GOOGLE_PROJECT_ID: {{ .Env.GCP_PRODUCTION_PROJECT }}
   release_environment: &release_environment
```

Comment:
We should probably rename this to `staging_environment` to avoid confusion. But seems better to handle separately as it would involve changing the config header file on all NROs.